app/v1/controllers/requestParsers/validators/DeleteOtherDeductionsValidator.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators
+
+import v1.controllers.requestParsers.validators.validations._
+import v1.models.errors.MtdError
+import v1.models.request.deleteOtherDeductions.DeleteOtherDeductionsRawData
+
+class DeleteOtherDeductionsValidator extends Validator[DeleteOtherDeductionsRawData] {
+
+  private val validationSet = List(parameterFormatValidation)
+
+  private def parameterFormatValidation: DeleteOtherDeductionsRawData => List[List[MtdError]] = (data: DeleteOtherDeductionsRawData) => {
+    List(
+      NinoValidation.validate(data.nino),
+      TaxYearValidation.validate(data.taxYear)
```

Comment:
Tax Year Validation still returns RULE_TAX_YEAR_RANGE_EXCEEDED instead of INVALID